lib/player.js

```diff
@@ -3449,7 +3451,9 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
       const variant = this.getPresentationVariant_();
 
       if (variant) {
-        this.stats_.setVariantBandwidth(variant.bandwidth);
+        const rate = this.playRateController_ ?
+           this.playRateController_.getRealRate() : 1;
+        this.stats_.setVariantBandwidth(rate * variant.bandwidth);
```

Comment:
I agree that the stats should show the current consumption of bandwidth, which would account for playbackRate.  But the docs for setVariantBandwidth should state this, to avoid internal mistakes, and for the app developer, the the docs for `streamBandwidth` in the stats should say explicitly that this accounts for playbackRate.

https://github.com/google/shaka-player/blob/cdbf8c5fbc7a4d4cb18e2fac4042e73317cfb757/externs/shaka/player.js#L91-L92

Reply:
I changed the description and the method name.